curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -7,11 +7,17 @@ dashedName: step-69
 
 # --description--
 
-One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` element as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
+You can set browser behavior by adding self-closing `meta` elements in the `head`. Here's an example:
+
+```
+<meta attribute="value" />
```

Comment:
Remove `/` as mentioned in the discussion

Reply:
```suggestion
<meta attribute="value">
```